driver/userspace.c

```diff
@@ -716,20 +716,20 @@ WnbdParseUserIOCTL(PWNBD_EXTENSION DeviceExtension,
             (PWNBD_IOCTL_STATS_COMMAND) Irp->AssociatedIrp.SystemBuffer;
 
         if (!StatsCmd || CHECK_I_LOCATION(IoLocation, WNBD_IOCTL_STATS_COMMAND)) {
-            WNBD_LOG_ERROR("WNBD_STATS: Bad input buffer");
+            WNBD_LOG_WARN("WNBD_STATS: Bad input buffer");
             Status = STATUS_INVALID_PARAMETER;
             break;
         }
 
         StatsCmd->InstanceName[WNBD_MAX_NAME_LENGTH - 1] = '\0';
         if (!strlen((PSTR) &StatsCmd->InstanceName)) {
-            WNBD_LOG_ERROR("WNBD_STATS: Invalid instance name");
+            WNBD_LOG_WARN("WNBD_STATS: Invalid instance name");
             Status = STATUS_INVALID_PARAMETER;
             break;
         }
 
         if (!Irp->AssociatedIrp.SystemBuffer ||
-                CHECK_O_LOCATION(IoLocation, WNBD_DRV_STATS)) {
+            CHECK_O_LOCATION(IoLocation, WNBD_DRV_STATS)) {
```

Comment:
Unnecessary change, now it's more difficult to see where the if statement ends.

Reply:
I will revert the change, although both seem terrible. I'm wondering if we should move the logical operator on a new line, that will help with the visibility.

Reply:
Yep, that would work as well.